### PR TITLE
Deactivate orphaned users when removed from last org

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
If a user only belongs to one org and is removed from that org (either from the admin page or the org member page) the user will be prompted to confirm that that will delete the user (rather than orphaning them).

Also:
- Fixes add/remove user form validation and console error
- Fixes a variety of small bugs on the admin page
- Fixes a bug with updating the default org for a user that was removed

#### How should this be manually tested?
1. Add a user to at least two orgs
2. Delete the user from those orgs and confirm that the last one prompts you to delete the user
#### What are the relevant tickets?
#994

### CODE REVIEW

File: docs/source/developer_resources.rst [hunk 1]
```diff
@@ -305,7 +305,7 @@ To make a release do the following:
     python docs/scripts/change_log.py –k GITHUB_API_TOKEN –s 2018-02-26 –e 2018-05-30
 
 4. Paste the results (remove unneeded Accepted Pull Requests) into the CHANGELOG.md. Make sure to cleanup the formatting.
-5. Make sure that any new UI needing localization has been tagged for translation, and that any new translation keys exist in the lokalise.co project. (see :doc:`translation documentation <translation>`).
+5. Make sure that any new UI needing localization has been tagged for translation, and that any new translation keys exist in the lokalise.com project. (see :doc:`translation documentation <translation>`).
```
Comment (nllong) [inline docs/source/developer_resources.rst]: i saw that they changed their URL... thanks for fixing this.

File: seed/views/organizations.py [hunk 2]
```diff
@@ -460,14 +464,15 @@ def remove_user(self, request, pk=None):
...
-            # assign them to a new organization if they want to use the account again.
-            first_org_user = OrganizationUser.objects.filter(user=user).order_by('id').first()
-            if first_org_user:
-                request.user.default_organization = first_org_user.organization
-                request.user.save()
+        user_orgs = OrganizationUser.objects.filter(user=user)
+
+        if user_orgs.count() == 0:
+            # Remove orphaned user
+            user.delete()
```
Comment (nllong) [inline seed/views/organizations.py]: are we sure we want to delete the user? Or should we just deactivate their account? If we deactivate the account, then we would need to make sure that the is_active flag is respected throughout the code. My main concern is that there are models that reference user ids (e.g., notes)

https://github.com/SEED-platform/seed/blob/4587ae1e8a1eec589677086ce1c887898521d13b/seed/landing/models.py#L53-L56

**Comment by nllong** [comment]
@axelstudios is this ready for review/testing?

**Comment by axelstudios** [comment]
@nllong Updated to deactivate users instead of deleting (with migration for existing org-less users)

**Pull request merged**